bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
Oh, clever! u' '.join works in both, converting py2k string-as-bytes to unicode, leaving py3k strings alone.